web/client/components/data/download/DownloadOptions.jsx

```diff
@@ -63,12 +63,19 @@ module.exports = class extends React.Component {
                     options={this.props.wfsFormats.map(f => ({value: f.name, label: f.label || f.name}))} />
             );
         }
+        const mergedOptions = this.props.wfsFormats
```

Comment:
hi i would call this availableWFSFormats 
```suggestion
        const availableWFSFormats = this.props.wfsFormats
```